MAINTAINERS.md

```diff
@@ -67,7 +67,7 @@ Update Jenkins job.
 ## Where to find the release binaries
 
 After Jenkins job for this release finished it will automaticaly post this tag on [GitHub release page](https://github.com/storj/storj/releases). The status will be `Draft`.
-Update this tag with changelog that you previosly created.
+Update this tag with changelog that you previosly created. Leave the status as draft as this will be updated after the storage node roll out is completed.
```

Comment:
are you sure it needs to be left as draft? Draft vs. published is a separate thing from release tagged as latest. In other words, you can still publish a new tag and _not_ mark it as "latest".

Reply:
It was listed as draft before this step so I imagine we want to leave it as is. There was never any mention of changing it. Although you do have a point, it should be verified that in some document, in one place in the process to include when to change the status.

Reply:
See the warning at the beginning of this confluence page https://storjlabs.atlassian.net/wiki/spaces/OPS/pages/138084357/Storagenode+rollout